css-brochure-form/css-brochure-form/styles.css

```diff
@@ -0,0 +1,57 @@
+body {
+  margin: 100px;
+  max-width: 500px;
+  max-height: 500px;
+}
+
+label {
+  margin: 10px;
+  text-align: left;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 500;
+  src: url('BrandonGrotesque-Medium.woff2') format('woff2');
+  font-stretch: condensed;
+}
+
+input,
+select {
+  margin-bottom: 15px;
+  border-color: grey;
+  border-radius: 7px;
+  padding: 8px;
+  width: 500px;
+  size: 100;
+  max-width: 30;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  src: url('BrandonGrotesque-Regular.woff2') format('woff2');
+}
+
+select {
+  background-color: rgb(243, 243, 243);
+}
+
+h2 {
+  text-align: center;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  src: url('BrandonGrotesque-Medium.woff2') format('woff2');
+  color: orangered;
+}
+
+button {
+  margin-left: 400px;
```

Comment:
😱 😱 😱 😱 😱 

WOAH, that's a lot of margin! Are you using this to center the element? If so, there is a less intensive way to do it. You can set a width on this element, then apply `margin: auto` to it. In this case, the margin value is being adapted on either side of the element based on how much available space there is. Though both solutions make use of margin, this form of doing so is less manual, and as such easier to adapt in the future, if necessary.